app/main/auto-updater/update-checker.js

```diff
@@ -0,0 +1,11 @@
+import request from 'request-promise';
+import { getFeedUrl } from './config';
+
+export async function getUpdate (currentVersion) {
+  const res = await request.get(getFeedUrl(currentVersion));
```

Comment:
I assume this may throw an exception in case of request failure

Reply:
Yep. Good catch. It should return false on error.